core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannel.scala

```diff
@@ -33,48 +33,41 @@ import scala.collection.{concurrent, immutable, mutable}
 import collection.JavaConverters._
 
 class TransactionMarkerChannel(interBrokerListenerName: ListenerName,
```

Comment:
ack. After thinking about this, I'm going to merge the `markerChannel` into `markerChannelManager` since it is always a one-to-one mapping.